src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -259,7 +259,8 @@ CHIP_ERROR ScenesServer::FabricSceneInfo::FindFabricSceneInfoIndex(EndpointId en
 {
     VerifyOrReturnError(kInvalidEndpointId != endpoint, CHIP_ERROR_INVALID_ARGUMENT);
 
-    uint16_t index = emberAfGetClusterServerEndpointIndex(endpoint, Scenes::Id, EMBER_AF_SCENES_CLUSTER_SERVER_ENDPOINT_COUNT);
+    uint16_t index =
+        emberAfGetClusterServerEndpointIndex(endpoint, MatterScenes::Id, EMBER_AF_SCENES_CLUSTER_SERVER_ENDPOINT_COUNT);
```

Comment:
est ce que `EMBER_AF_SCENES_CLUSTER_SERVER_ENDPOINT_COUNT` est censé changer aussi?

Reply:
It doesn't _need_ to as it is still generated as `EMBER_AF_SCENES_CLUSTER_SERVER_ENDPOINT_COUNT` in gen_config, the question is, do we want to change it?

Reply:
After consideration, minimizing the changes seems like the better idea so we will keep the define as is for now.